[CMake] Wrong message about dependency between introspection and gi-docgen

siwei-li opened the pull request:
#### 85864ba4b06927ed06f9aec10807f642510adde0
<pre>
[CMake] Wrong message about dependency between introspection and gi-docgen
<a href="https://bugs.webkit.org/show_bug.cgi?id=242710">https://bugs.webkit.org/show_bug.cgi?id=242710</a>

Reviewed by Adrian Perez de Castro.

* Source/cmake/OptionsGTK.cmake: Fix the error message where introspection is needed
for the gi-docgen documentation generator.
* Source/cmake/OptionsWPE.cmake: Same as above.

Canonical link: <a href="https://commits.webkit.org/252538@main">https://commits.webkit.org/252538@main</a>
</pre>

aperezdc reviewed (approved):
@siwei-li: Thanks for the patch!

The EWS failures are unrelated to this change, so let's land it.

siwei-li committed:
[CMake] Wrong message about dependency between introspection and gi-docgen
https://bugs.webkit.org/show_bug.cgi?id=242710

Reviewed by Adrian Perez de Castro.

* Source/cmake/OptionsGTK.cmake: Fix the error message where introspection is needed
for the gi-docgen documentation generator.
* Source/cmake/OptionsWPE.cmake: Same as above.

Canonical link: https://commits.webkit.org/252538@main
